openedx/core/djangoapps/content/learning_sequences/admin.py

```diff
@@ -6,15 +6,112 @@
 import json
 
 from django.contrib import admin
-from django.utils.html import format_html
+from django.utils.html import format_html, format_html_join
+from django.utils.translation import gettext_lazy as _
 from opaque_keys import OpaqueKey
 import attr
 
-from .api import get_course_outline
-from .models import LearningContext
+from .api import get_content_errors, get_course_outline
+from .models import CourseContext, CourseSectionSequence
 
 
-class LearningContextAdmin(admin.ModelAdmin):
```

Comment:
just checking my understanding here -- are you switching out LearningContextAdmin for CourseContextAdmin because the new view is exposing course-specific details, like Sections?

Reply:
Yes. Technically, it would have worked with LearningContext as well, but the admin functionality is starting to delve into things that are really course-specific.